src/hadoop-resource-manager/deploy/hadoop-resource-manager.yaml.template

```diff
@@ -45,6 +45,8 @@ spec:
           name: yarn-resourcemanager-storage
         - mountPath: /var/lib/hadoopdata
           name: hadoop-tmp-storage
+        - mountPath: /host-configuration
```

Comment:
why do you this (and the configmap)?

Reply:
One machine may have several eth and serveral ip-address. The host-configuration has a yaml containing the machine-list. We can grep the file to find the ip of the machine which configured in our configuration